experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -600,6 +600,65 @@ export interface Sequence<in out TTypes extends AllowedTypes> extends TreeField
 	 */
 	removeRange(start?: number, end?: number): void;
 
+	/**
+	 * Moves the specified item to the start of the sequence.
+	 * @param sourceIndex - The index of the item to move.
```

Comment:
Would be nice to define what makes a valid index. E.g., do we support negative indices?

Reply:
Clarified, thanks.